README.md

```diff
@@ -60,6 +60,15 @@ When your environment cannot access `pypi`, you need to install the following pa
 - httplib2
 - futures   # python2.x is needed
```

Comment:
line 57-61 can be deleted, directly to use requirements.txt is ok.

Reply:
Done